mne/decoding/transformer.py

```diff
@@ -170,7 +170,7 @@ def transform(self, epochs_data, y=None):
         memory usage may be large with big data.
         """
         return _sklearn_reshape_apply(self._scaler.transform, True,
-                                      epochs_data, y)
+                                      epochs_data)
```

Comment:
@kingjr is this change okay for you? It's giving errors with `sklearn` 0.19. The `Scaler.transform` shouldn't need the labels, right?